docs/data/tree-view/simple-tree-view/customization/CustomContentTreeView.tsx

```diff
@@ -1,93 +1,72 @@
 import * as React from 'react';
```

Comment:
I migrated this example to the new API to give an example of what it would look like

**BUT** we should not merge it like this, we should keep the old example until the new API is public